packages/frontend-shared/src/utils/spec-utils.ts

```diff
@@ -4,8 +4,20 @@ import { ComputedRef, Ref, ref, watch } from 'vue'
 import _ from 'lodash'
 
 import type { UseCollapsibleTreeNode } from '../composables/useCollapsibleTree'
+import { decodeBase64Unicode } from './decodeBase64'
 
-const PATH_SEP = /[\/\\]/
+const getPlatform = () => {
+  // @ts-ignore
+  return window.Cypress?.platform || JSON.parse(decodeBase64Unicode(window.__CYPRESS_CONFIG__.base64Config)).platform
```

Comment:
This seems to be necessary since the specs list exists before Cypress necessarily exists